src/affordance/tasks/arxiv_summarization.py

```diff
@@ -0,0 +1,384 @@
+import argparse
```

Comment:
I actually think it's better to have a PR contain just one new thing at a time. Better to leave Georges' stuff in his branch